autovacuum_mail_message/models/mail_message.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018 Akretion
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import api, models
+import openerp
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class MailMessage(models.Model):
+    _inherit = "mail.message"
+
+    @api.multi
+    def batch_unlink(self):
+        with api.Environment.manage():
```

Comment:
Merge the `with` using comma
I mean,
```python
with api..., openerp...
```

Reply:
Is it mandatory, I think it is harder to read...

Reply:
I guess not. matter of taste on that point.